cmake-docker-environment/lab-files/targets-example/CMakeLists.txt

```diff
@@ -1,13 +1,12 @@
 cmake_minimum_required(VERSION 3.9.1)
 project(songTest VERSION 1.0.0)
 
-add_library(firstDay SHARED firstDay/firstDay.cpp)
+add_library(firstDay firstDay/firstDay.cpp)
+target_include_directories(firstDay PUBLIC ${CMAKE_CURRENT_LIST_DIR}/firstDay/include)

```

Comment:
For libraries, I normally include only the header files related to that library, but not sure if this is a good practice.